pyproject.toml

```diff
@@ -54,6 +54,7 @@ svgelements = ">=1.8.0"
 tqdm = ">=4.0.0"
 typing-extensions = ">=4.0.0"
 watchdog = ">=2.0.0"
+av = "^11.0.0"
```

Comment:
Put this at the top of the deps (after `python`) so we keep them ordered?

Reply:
Yes, and probably also make the version restriction less strict (cc @Viicos)

Reply:
Regarding the "less strict" Python version, I think this is a general issue with Poetry where if a dependency you need requires some Python version, you then have to explicitly specify a Python constraint.

Reply:
Yes `av>=some_version` should be ok. Looking at the [changelog](https://github.com/PyAV-Org/PyAV/blob/main/CHANGELOG.rst), it seems pretty stable so we can probably go for `>=9.0.0` or something? This can be bumped of course if you need some features in higher versions.

> Regarding the "less strict" Python version, I think this is a general issue with Poetry where if a dependency you need requires some Python version, you then have to explicitly specify a Python constraint.

I think @behackl was talking about the constraint on `av`. But yeah regarding the Python version we unfortunately had to add a higher bound

Reply:
@Viicos got it :p